application/common/models/User.php

```diff
@@ -48,29 +49,59 @@ public function rules()
     }
 
     /**
+     * Limit what fields are returned from api calls
+     * @return array
+     */
+    public function fields()
+    {
+        return [
+            'first_name',
+            'last_name',
+            'idp_username' => function($model) {
+                return strtolower($model->idp_username);
```

Comment:
Yeah good point, I was mainly thinking that this data is returned to the UI/client and would be displayed and being all caps is ugly. But I could leave it as is and let the UI/client format it however they want.